frontend/public/components/chargeback.tsx

```diff
@@ -300,10 +286,11 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
       dataElem = <LoadError label="Report" message={_.get(error, 'json.error') || error.message} />;
     } else if (data) {
       if (data.error) {
-        dataElem = <LoadError label="Report" message={data.error} />;
+        if (data.error !== 'Report is not processed yet') {
```

Comment:
It does happen and the user experience isn't that good because it shows a red error of 'Report is not processed yet', when IMO that's not an error.  Adding this skips this and uses the default `dataElem = <MsgBox title="No Data" detail="Report not finished running." />;` which is what I would expect we would want to show the user.

Reply:
Maybe we remove the error styling if it’s not always an error. Checking for an exact message is fragile and likely to break in the future

Reply:
Is there a reason code or anything else in the error response we can check?

Reply:
This doesn't appear to be happening anymore.  I manually installed the latest operator-metering this morning (as opposed to OLM install), and I don't get this intermittent 'Report is not processed yet' error, so I have removed this check.